Reorganising file structure to be easier to navigate sidebar (#78)
lukso-network/website-support-center: docs/general/getting-started/getting-started.md

---
sidebar_label: '🚀 Getting Started'
sidebar_position: 1
description: 'LUKSO Support: Universal Profile Controllers'
---

# Getting Started

This section is aimed at getting your from zero knowledge --> interacting with dapps on the LUKSO blockchain.

**The guide is broken down into 4 sections**:

1. Creating your Universal Profile
2. Downloading our 🆙 browser extension
3. Buying some LYX

> 4. Sign posting your to some popular dapps on our network

Reviewer: ```suggestion
4. Sign posting you to some popular dapps on LUKSO
```
